### ISSUE TITLE
Example of SpecFlow.Actions.Selenium with Selenium Grid 4

### ISSUE THREAD

**Issue by dfibuch** [SpecFlowOSS/SpecFlow.Actions#73]
Does anyone have an example how to make [this](https://github.com/SpecFlowOSS/SpecFlow-Examples/tree/master/Actions/SeleniumExample) work with Selenium Grid 4? I'm fairly new to using Selenium so still getting my head around it but I've been using SpecFlow for a while (C# integration tests for an API).

I understand how SpecFlow and Selenium work, I've played around with the example to hit one of my sites and do a basic test but what I don't get is how the Grid fits in to this, how I'm supposed to pass in the Grid Url AND my site Url.

Any help greatly appreciated.

**Comment by nealio42** [comment]
Hi, I am also interested in this. Thanks.

**Comment by nealio42** [comment]
I don't think this can be done in the current implementations. We would need RemoteDriver here.

**Comment by dfibuch** [comment]
> I don't think this can be done in the current implementations. We would need RemoteWebDriver here. Which makes this unusable. Bummer.

Ye I started looking into it and having a play before realising there is something missing

**Comment by SabotageAndi** [comment]
The Selenium Action doesn't have yet support for the RemoteWebDriver. But it should not be hard to add.
Anybody wants to send a PR?
